apps/condo/domains/acquiring/tasks/recurrent-payment-context-processing-task.js

```diff
@@ -0,0 +1,7 @@
+const { createCronTask } = require('@open-condo/keystone/tasks')
+
```

Comment:
it looks like redundant file

Reply:
Moved to implementation file (as I can see for other tasks)